src/java.base/windows/classes/java/io/WinNTFileSystem.java

```diff
@@ -490,12 +490,20 @@ public String canonicalize(String path) throws IOException {
                 return path;
             return "" + ((char) (c-32)) + ':' + '\\';
         }
-        return canonicalize0(path);
+        String canonicalPath = canonicalize0(path);
+        return getFinalPath(canonicalPath);
```

Comment:
This needs to be changed to ignore any error in `getFinalPath` and return `canonicalPath` instead.

Reply:
So changed in 2d6423b.